### ISSUE TITLE
Re-assign records to the correct boundaries

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1069]
A few months ago @johnvanbreda re-assigned some records that had got linked to incorrect location details. The records are now displaying in the correct place, but are still showing the incorrect vice-counties and LERCs etc. I assume they need re-indexing (but at least some have been verified, so ideally should not be triggered for re-verification).

The record IDs concerned are given below.
`occurrence ID IN (212576, 4678865, 4678868, 4678867, 4678800, 4678842, 4678759, 4678808, 4678788, 4678836, 4678763, 4678807, 4678816, 4678841, 4678799, 4678789, 4678764, 4678823, 4678762, 4678791, 4678780, 4678761, 4678782, 4678824, 4678781, 4678770, 4678790, 4678812, 4678793, 4678835, 4678778, 4678838, 4678796, 4678766, 4678853, 4678856, 4678795, 4678845, 4678832, 4678844, 4678758, 4678849, 4678813, 4678861, 4678815, 4678786, 4678826, 4678765, 4678850, 4678798, 4678805, 4678839, 4678801, 4678767, 4678817, 4678818, 4678819, 4678820, 4678822, 4678825, 4678851, 4678760, 4678768, 4678779, 4678787, 4678792, 4678794, 4678797, 4678837, 4678863, 4678769, 4678771, 4678772, 4678773, 4678774, 4678775, 4678776, 4678777, 4678783, 4678784, 4678785, 4678802, 4678803, 4678804, 4678806, 4678810, 4678811, 4678814, 4678827, 4678828, 4678829, 4678830, 4678831, 4678833, 4678834, 4678840, 4678843, 4678846, 4678847, 4678848, 4678852, 4678854, 4678855, 4678857, 4678858, 4678859, 4678860, 4678862, 4678864, 4678866)`

**Comment by kitenetter** [comment]
At least two of the above records (IDs 4678867 and 4678868) have an additional problem in that I can find them in the warehouse but I cannot get them to display on iRecord; and also they have Tim Arnold's name attached as determiner, which is erroneous.

I just tried to edit them via the warehouse but it timed out.

**Comment by johnvanbreda** [comment]
I've requeued these records for re-indexing and rebuilding the attribute data which should clean things up. Looking at the audit trail I can see that Tim Arnold was assigned as the determiner but this was updated and the update hadn't refreshed in attrs_json. In this particular case as as least some manual manipulation has occurred, it's likely that user error (i.e. me!) is the cause.

Those records don't show in iRecord because they are training records.
